src/js/app.controller.js

```diff
@@ -24,20 +24,12 @@
     $scope.fileSearch = null;
     $scope.allFiles = {};
 
-    // vm.create = () => {
-    //   let result = vm.indexer.test();
-    //   alert(result);
-    // };
-
     vm.create = () => {
```

Comment:
### Nitpick
Adopt best practice for naming variables or objects. `vm` is not descriptive.